components/styles.css

```diff
@@ -6,13 +6,17 @@
   border-radius: 5px;
   border-color: var(--background-tertiary);
   background-color: var(--background-secondary);
+}
 
-  .vc-friend-codes-card-title span {
-    color: var(--header-secondary);
-    font-family: var(--font-primary);
-    font-size: 14px;
-    font-weight: 400;
-  } 
+.vc-friend-codes-card-title :first-child {
+  text-transform: none;
+}
```

Comment:
Please add the Rule in the [FormTitle](https://github.com/Sqaaakoi/vc-FriendCodes/blob/85a196d5a169d035bb7b5374c9d9e588e0995322/components/FriendCodesPanel.tsx#L35) directly, rather than having it in the CSS File
```js
<Forms.FormTitle tag="h4" style={{ textTransform: "none"}}>
```